src/frontend/src/binder/insert.rs

```diff
@@ -26,6 +27,7 @@ use crate::binder::{Binder, Clause};
 use crate::catalog::TableId;
 use crate::error::{ErrorCode, Result, RwError};
 use crate::expr::{ExprImpl, InputRef};
+use crate::handler::create_mv::ordinal;
```

Comment:
Shall we move it out of the module?

Reply:
Wanted to minimize diff. Can do it before merge.